jupyterlab_bigquery/src/components/query_editor/query_editor_incell/query_editor_incell.tsx

```diff
@@ -40,19 +41,26 @@ export class QueryEditorInCell extends Component<QueryEditorInCellProps, {}> {
     this.props.ipyView.touch();
 
     return (
-      <div style={{ width: '75vw' }}>
-        <QueryTextEditor
-          queryId={this.queryId}
-          iniQuery={this.iniQuery}
-          editorType="IN_CELL"
-          queryFlags={this.queryFlags}
-        />
-        {showResult ? (
-          <QueryResults queryId={this.queryId} editorType="IN_CELL" />
-        ) : (
-          undefined
-        )}
-      </div>
+      <ReactResizeDetector>
+        {({ width }) => {
+          return (
+            <div>
+              <QueryTextEditor
+                queryId={this.queryId}
+                iniQuery={this.iniQuery}
+                editorType="IN_CELL"
+                queryFlags={this.queryFlags}
+                width={width}
+              />
+              {showResult ? (
+                <QueryResults queryId={this.queryId} editorType="IN_CELL" />
+              ) : (
+                undefined
+              )}
+            </div>
+          );
+        }}
+      </ReactResizeDetector>
```

Comment:
Similar to `UseSignal`, `ReactResizeDetector` will rerender its children whenever the width changes and is passed down. There is also a HOC available, but I thought this was more similar to how we resize the full-window editor.